nsxt/resource_nsxt_policy_lb_pool_test.go

```diff
@@ -26,6 +27,7 @@ var accTestPolicyLBPoolUpdateAttributes = map[string]string{
 	"algorithm":               "WEIGHTED_ROUND_ROBIN",
 	"min_active_members":      "5",
 	"tcp_multiplexing_number": "5",
+	"active_monitor_path":     "/infra/lb-monitor-profiles/default-http-lb-monitor",
```

Comment:
Lets also validate that this attribute is assigned within the test (validating that list size `active_monitor_paths.#` equals to 1)

Reply:
right, I forgot to do that